src/redux/store.js

```diff
@@ -0,0 +1,5 @@
+import { configureStore } from "@reduxjs/toolkit";
+
+export const store = configureStore({
+  reducer: {},
+})
```

Comment:
- [ ] Well done creating the reducers. Now you have to import them to your configureStore for them to be usable. Kindly add the categorySlice and the bookSlice to your reducer.